frontend/packages/ux-editor/src/utils/component.ts

```diff
@@ -28,3 +30,43 @@ export enum AddressKeys {
   careOf = 'careOf',
   houseNumber = 'houseNumber',
 }
+
+export const changeTextResourceBinding = (
+  component: IFormComponent,
+  bindingKey: string,
+  resourceKey: string
+): IFormComponent => ({
+  ...component,
+  textResourceBindings: {
+    ...component.textResourceBindings,
+    [bindingKey]: resourceKey,
+  }
+});
+
+export const changeTitleBinding = (component: IFormComponent, resourceKey: string): IFormComponent =>
+  changeTextResourceBinding(component, 'title', resourceKey);
+
+export const changeDescriptionBinding = (component: IFormComponent, resourceKey: string): IFormComponent =>
+  changeTextResourceBinding(component, 'description', resourceKey);
+
+export const addOptionToComponent = <T extends IFormCheckboxComponent | IFormRadioButtonComponent>(
+  component: T,
+  option: IOptions
```

Comment:
Is this the right type for a single option?

Reply:
I seems like somebody just added an s by error. I'll see if I can fix it.